docs/how-to-guides/block-tutorial/nested-blocks-inner-blocks.md

```diff
@@ -155,13 +155,15 @@ add_action( 'init', function() {
 } );
 ```
 
-## Child InnerBlocks: Parent and Ancestors
+## Using Hierarchical Context in InnerBlocks
 
-A common pattern for using InnerBlocks is to create a custom block that will be included only in the InnerBlocks.
+A common pattern for using InnerBlocks is to create a custom block that will be included _only_ in the InnerBlocks. This allows InnerBlocks to have a hierarchical context for nesting child blocks.
 
-An example of this is the Columns block, that creates a single parent block called `columns` and then creates an child block called `column`. The parent block is defined to only allow the child blocks. See [Column code for reference](https://github.com/WordPress/gutenberg/tree/HEAD/packages/block-library/src/column).
+### Defining Parent Block Context
 
-When defining a child block, use the `parent` block setting to define which block is the parent. This prevents the block showing in the inserter outside of the InnerBlock it is defined for.
+An example of this is the Column block, which is assigned the `parent` block setting. This allows the Column block to only be available in its parent Columns block context. Otherwise, the Column block will not be available as an option within the block inserter. See [Column code for reference](https://github.com/WordPress/gutenberg/tree/HEAD/packages/block-library/src/column).
```

Comment:
```suggestion
An example of this is the Column block, which is setting the `parent` block setting to `[ "core/columns" ]`. This allows the Column block to only be available as a direct descendant of the Columns block. Otherwise, the Column block will not be available within the block inserter. See [Column code for reference](https://github.com/WordPress/gutenberg/tree/HEAD/packages/block-library/src/column).
```